docs/research/2026-07-22-direct-ptx-optimizer-blueprint.md

```
@@ -0,0 +1,100 @@
+Date: 2026-07-22
+
+Tracking issue: #848
+
+Parent blueprint: `2026-07-20-fused-attention-championship-blueprint.md`
+
+## Verdict
+
+This increment establishes the issue-#848 assembly line and implements one exact
+contiguous Ampere FP32 candidate: a **fused SGD-with-momentum update step**:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)